packages/mobile/src/web3/saga.ts

```diff
@@ -55,10 +56,18 @@ export const SYNC_TIMEOUT = 2 * 60 * 1000 // 2 minutes
 const BLOCK_CHAIN_CORRUPTION_ERROR = "Error: CONNECTION ERROR: Couldn't connect to node on IPC."
 const SWITCH_TO_FORNO_TIMEOUT = 15000 // if syncing takes >15 secs, suggest switch to forno
 const WEB3_MONITOR_DELAY = 100
+// track sync time first time and then only one every so many events.
+const TRACK_SAMPLING_PERIOD_SYNC = 1
+
+// Tracking meanMilliseconds and count per analytics event that need sampling (too frequent).
+const performanceTracking = {
```

Comment:
I believe you can use `CeloAnalytics.startTracking` + `trackSubEvents` + `stopTracking` to serve this purpose